ct/adventurelog.sh

```diff
@@ -66,6 +71,7 @@ function update_script() {
 
     msg_info "Cleaning Up"
     rm -rf v${RELEASE}.zip
+    mv /opt/adventurelog-backup/ adventurelog-backup-$LAST_VERSION/ # Keep backup, since AdventureLog moves quickly and we want to avoid breaking changes with rm -rf
```

Comment:
```suggestion
    rm -rf /opt/adventurelog-backup
```
We do not want to be responsible for user data backups. This should be done with lxc updates

Reply:
@michelroegl-brunner This is not directly about user backup. It's rather the thought that you, as maintainer of this community script, need to either be completely on top of changes within AdventureLogs that might introduce additional files that a user generates. Or you keep old versions after an update (or the secret third option, make some intelligent check).

This is not so much a backup, but rather an insurance policy that during an update you don't delete surprise files from a user (keep them retrievable).

Reply:
I´m working on more "global" approach for this issue atm. It will take a little bit of time. Should be a mix of your option " and 3.

Reply:
I'm quite nervous about entirely removing the old data. But if you say this will be fixed more elegantly, I'll put the responsibility on your shoulders. You're the maintainer, after all.

Reply:
yes, the plan is to create a tarball everytime a update is done and store it at the lxc itself, so no user data will get lost. And also we can then revert changes if a update fails.